alpakaConfig.cmake

```diff
@@ -466,6 +466,10 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                     IF(CUDA_VERSION GREATER_EQUAL 10.1)
                         MESSAGE(FATAL_ERROR "Clang versions lower than 9 do not support CUDA 10.1 or greater!")
                     ENDIF()
+                ELSEIF(CMAKE_CXX_COMPILER_VERSION LESS 10.0)
+                    IF(CUDA_VERSION GREATER_EQUAL 10.2)
+                        MESSAGE(FATAL_ERROR "Clang versions lower than 10 do not support CUDA 10.2 or greater!")
```

Comment:
In the updated readme the table states that cuda 10.1 - 10.2 is supported for clang 4 - 8. Seems to contradict this error message.

Reply:
Same slightly upper with clang 9 and cuda 10.1

Reply:
Yes, the ReadMe is wrong. nvcc, 10.2 + clang 9 is not supported.

Reply:
After looking at it again, everything seems to be correct. Those checks here are for the clang native CUDA support, not about the clang versions supported by nvcc.
What we check in the previous condition is that clang 1 to 8 do not support CUDA 10.1+ which is true.
In the marked line we say that clang 1 to 9 do not support CUDA 10.2+ which is also true.
We did not change anything in the readme for clangs supported CUDA versions.

Reply:
Ah, my bad, thanks for clarifying.